auth/README.md

```diff
@@ -0,0 +1,69 @@
+# Auth Package
+
+Auth package is used for request authentication and authorization for user to service
```

Comment:
Instead of "Auth package is used for…" you could say more directly, "Use package auth for…" or even "Package auth authenticates and authorizes requests for user-to-service…."